internal/masterservice/meta_table.go

```diff
@@ -232,7 +228,8 @@ func (mt *metaTable) AddProxy(po *pb.ProxyMeta) error {
 	return nil
 }
 
-func (mt *metaTable) AddCollection(coll *pb.CollectionInfo, part *pb.PartitionInfo, idx []*pb.IndexInfo) error {
+func (mt *metaTable) AddCollection(coll *pb.CollectionInfo, part *pb.PartitionInfo, idx []*pb.IndexInfo,
+	collReq *internalpb.CreateCollectionRequest, partReq *internalpb.CreatePartitionRequest) error {
```

Comment:
please construct the `DdOperation` out of this function, not pass `collReq` and `partReq` into this function